server/games/custom_game.py

```diff
@@ -1,4 +1,3 @@
-import asyncio
```

Comment:
and why this was not marked as unused import before?

Reply:
Actually I am seeing an unused import warning for this using flake8. It might have been in there for a long time though, like since before we added codacy?